Keas.Mvc/Controllers/Api/AccessController.cs

```diff
@@ -192,6 +192,22 @@ public async Task<IActionResult> Update([FromBody] Access access)
 
         }
 
+        [HttpPost("{id}")]
+        [ProducesResponseType(typeof(AccessAssignment), StatusCodes.Status200OK)]
+        public async Task<IActionResult> UpdateAssignment([FromBody] AccessAssignment accessAssignment)
```

Comment:
@wongband see if you can use the existing `Assign` method here, like we do in equipment or other places. See https://github.com/ucdavis/Peaks/blob/master/Keas.Mvc/Controllers/Api/EquipmentController.cs#L210.  You'll probably have to add an IF statement to see if it's an existing assignment or just an update.  Also, we'll want to track this action like we do when the assignment is first updated.  Again check out the equipment controller a good example.